src/components/profile/myLibrariesPage/Page.js

```diff
@@ -12,34 +17,56 @@ import Button from "@/components/base/button";
  */
 
 export default function MyLibrariesPage() {
+  const { isAuthenticated } = useUser();
+  const [showMore, setShowMore] = useState(false);
+  const { data: userData } = useData(
+    isAuthenticated && userFragments.branchesForUser()
+  );
+
+  const result = userData?.user?.agency?.result;
+
+  //Find a list of user agencies
+  const agencies = [];
+  const addedAgencyIds = [];
+
+  result?.forEach((branch) => {
+    if (!addedAgencyIds.includes(branch.agencyId)) {
+      addedAgencyIds.push(branch.agencyId);
+      agencies.push({
+        agencyId: branch.agencyId,
+        agencyName: branch.agencyName,
+      });
+    }
+  });
```

Comment:
Fejler denne ordentligt hvis Branch f.eks. mangler `agencyId` og `agencyName`.

Hvad hvis den har `agencyName` men ikke `agencyId`? Så kommer der muligvis en off-by-1 i relationen mellem `addedAgencies` og `agencies` (jeg ved dog ikke om det er et problem

Reply:
Yes, godt set. Har lavet et fix så der tages højde for hvis agencyid eller name mangler. Tænkte bare at der ikke var nogle tilfælde hvor de to værdier er null.